model/Software/Properties/packageUrl.md

```diff
@@ -9,30 +9,26 @@ Provides a place for the SPDX data creator to record the package URL string
 
 ## Description
 
-A package URL (commonly pronounced and referred to as "purl") is an attempt to
-standardize package representations in order to reliably identify and locate
-software packages. A purl is a URL string which represents a package in a
+A package URL is an attempt to standardize package representations
```

Comment:
```suggestion
A package URL (commonly pronounced and referred to as "purl") is an attempt to standardize package representations
```

Reply:
This is already in the Annex defining PURL, but I can add it back if you insist.

You know that I am not very font of this, especially when our RDF uses `purl.org` which means something else...

Reply:
If we don't have it,  it will open up questions, so rather keep the wording that's already been debated, rather than change it at this last minute.